src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './SearchBar.scss';
+
+export default class SearchBar extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { value: '' };
+        this.handleChange = this.handleChange.bind(this);
+        this.onSubmit = this.onSubmit.bind(this);
+    }
+
+    onSubmit(event) {
+        const { handleSubmit } = this.props;
+        const {
+            value,
+        } = this.state;
+        handleSubmit(value);
+        event.preventDefault();
```

Comment:
place before handleSubmit(value);
need to add empty lines to make code more readable